gke-debian-openjdk/src/main/docker/Dockerfile

```diff
@@ -17,44 +17,25 @@ ENV DEBIAN_FRONTEND noninteractive
 
 # Update debian 
 RUN echo 'deb http://httpredir.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list \
- && apt-get -q update \
- && apt-get -y -q --no-install-recommends install \
-    ca-certificates \
-    openjdk-8-jre \
+  && apt-get -q update \
+  && apt-get -y -q --no-install-recommends install \
+    ca-certificates-java=20161107'*' \
+    openjdk-8-jre-headless=8u121'-*' \
```

Comment:
I don't think you need to specify `8u121` version here. It will pick up latest anyway.